<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>997</width>
    <height>631</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>49</y>
      <width>1001</width>
      <height>521</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>30</number>
    </attribute>
    <attribute name="verticalHeaderMinimumSectionSize">
     <number>30</number>
    </attribute>
    <column>
     <property name="text">
      <string>상태</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>경로</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>기존 파일이름</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>새 파일이름</string>
     </property>
    </column>
   </widget>
   <widget class="QCommandLinkButton" name="convertButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Convert</string>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>0</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Add Files</string>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="formatButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>0</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Set Format</string>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="pathButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>0</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Set Path</string>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>997</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionAdd_Files"/>
    <addaction name="separator"/>
    <addaction name="actionExit_2"/>
   </widget>
   <widget class="QMenu" name="menuSetting">
    <property name="title">
     <string>Setting</string>
    </property>
    <addaction name="actionString_Format"/>
    <addaction name="actionPath_Setting"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSetting"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAdd_Files">
   <property name="text">
    <string>Add Files</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionExit_2">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionString_Format">
   <property name="text">
    <string>String Format</string>
   </property>
  </action>
  <action name="actionPath_Setting">
   <property name="text">
    <string>Path Setting</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>convertSlot()</slot>
  <slot>addSlot()</slot>
  <slot>formatSlot()</slot>
  <slot>pathSlot()</slot>
 </slots>
</ui>
